package.json

```diff
@@ -21,14 +21,15 @@
     "lint:all": "npx eslint ./azure-functions/acbs-function && npx eslint ./azure-functions/number-generator-function && npx eslint ./dtfs-central-api && npx eslint ./e2e-tests && npx eslint ./gef-ui && npx eslint ./portal && npx eslint ./portal-api && npx eslint ./external-api && npx eslint ./trade-finance-manager-api && npx eslint ./trade-finance-manager-ui && npx eslint ./utils/mock-data-loader && npx eslint ./utils/data-migration",
     "lint:fix:all": "npx eslint --fix ./azure-functions/acbs-function && npx eslint --fix ./azure-functions/number-generator-function && npx eslint --fix ./dtfs-central-api && npx eslint --fix ./e2e-tests && npx eslint --fix ./gef-ui && npx eslint --fix ./portal && npx eslint --fix ./portal-api && npx eslint --fix ./external-api && npx eslint --fix ./trade-finance-manager-api && npx eslint --fix ./trade-finance-manager-ui && npx eslint --fix ./utils/mock-data-loader && npx eslint --fix ./utils/data-migration",
     "prepare": "husky install",
+    "spellcheck": "cspell lint --gitignore --no-must-find-files --unique --no-progress --show-suggestions --color '**/*'",
```

Comment:
I think it's going to find a lot of spelling issues in old files (until now, we've only been spell-checking files that we've edited) :)

Reply:
Yep around ~1,700 issues I need to resolve.